pgmpy/tests/test_inference/test_Mplp.py

```diff
@@ -0,0 +1,100 @@
+        factor_m_n = Factor(['M', 'N'], cardinality=[2, 2], value=np.array([0.014559, 68.687, 68.687, 0.014559]))
+        factor_n_o = Factor(['N', 'O'], cardinality=[2, 2], value=np.array([20.155, 0.049615, 0.049615, 20.155]))
+        factor_o_p = Factor(['O', 'P'], cardinality=[2, 2], value=np.array([0.013435, 74.435, 74.435, 0.013435]))
+        factor_a_e = Factor(['A', 'E'], cardinality=[2, 2], value=np.array([16.14, 0.061959, 0.061959, 16.14]))
+        factor_e_i = Factor(['E', 'I'], cardinality=[2, 2], value=np.array([0.001312, 762.21, 762.21, 0.001312]))
+        factor_i_m = Factor(['I', 'M'], cardinality=[2, 2], value=np.array([0.00099042, 1009.7, 1009.7, 0.00099042]))
+        factor_b_f = Factor(['B', 'F'], cardinality=[2, 2], value=np.array([352.33, 0.0028383, 0.0028383, 352.33]))
+        factor_f_j = Factor(['F', 'J'], cardinality=[2, 2], value=np.array([19.263, 0.051912, 0.051912, 19.263]))
+        factor_j_n = Factor(['J', 'N'], cardinality=[2, 2], value=np.array([141.3, 0.007077, 0.007077, 141.3]))
+        factor_c_g = Factor(['C', 'G'], cardinality=[2, 2], value=np.array([0.00023442, 4265.9, 4265.9, 0.00023442]))
+        factor_g_k = Factor(['G', 'K'], cardinality=[2, 2], value=np.array([134.43, 0.0074387, 0.0074387, 134.43]))
+        factor_k_o = Factor(['K', 'O'], cardinality=[2, 2], value=np.array([0.00015823, 6320.0, 6320.0, 0.00015823]))
+        factor_d_h = Factor(['D', 'H'], cardinality=[2, 2], value=np.array([1994.0, 0.0005015, 0.0005015, 1994.0]))
+        factor_h_l = Factor(['H', 'L'], cardinality=[2, 2], value=np.array([0.022576, 44.295, 44.295, 0.022576]))
+        factor_l_p = Factor(['L', 'P'], cardinality=[2, 2], value=np.array([0.0018291, 546.72, 546.72, 0.0018291]))
+
+        self.markov_model.add_factors(factor_a, factor_b, factor_c, factor_d,
+                                      factor_e, factor_f, factor_g, factor_h,
+                                      factor_i, factor_j, factor_k, factor_l,
+                                      factor_m, factor_n, factor_o, factor_p,
+                                      factor_a_b, factor_b_c, factor_c_d, factor_e_f,
+                                      factor_f_g, factor_g_h, factor_i_j, factor_j_k,
+                                      factor_k_l, factor_m_n, factor_n_o, factor_o_p,
+                                      factor_a_e, factor_e_i, factor_i_m, factor_b_f,
+                                      factor_f_j, factor_j_n, factor_c_g, factor_g_k,
+                                      factor_k_o, factor_d_h, factor_h_l, factor_l_p)
+
+        for factor in self.markov_model.factors:
+            factor.values = np.log(factor.values)
+        self.mplp = Mplp(self.markov_model)
+
+    def test_query_single_variable(self):
+        query_result = self.mplp.map_query(1000, 0.0002, 0.0002)
+        # Results from the Sontag code for a mplp run without tightening is:
+        sontag_result = {
```

Comment:
Don't use the term ```sontag_result``` over here. Instead rename it to ```expected_result``` and put the reference of the paper from where you got these values.